tests/lang.sh

```diff
@@ -1,5 +1,46 @@
 source common.sh
 
+set -o pipefail
+
+# Flip which one is commented to update
+
+#doAccept=1
+doAccept=''
```

Comment:
Maybe this should at least take an environment variable so it can be run slightly more conveniently?
Also add instructions to the hacking guide on how to deal with these tests in general, i.e.
- what to do when you change error messages?
- how to communicate these changes in PRs?

Reply:
Add instruction to update the `.err` to the error message when `.err` didn't match?
That makes the iteration cycle easier, and we can keep the hacking instructions simple as well.